client/client-multi/client-c/src/kaa/kaa_notification_manager.c

```diff
@@ -535,6 +559,7 @@ kaa_error_t kaa_add_notification_listener(kaa_notification_manager_t *self, kaa_
         KAA_LOG_WARN(self->logger, err, "Failed to calculate mandatory listener id");
         return err;
     }
+    KAA_LOG_TRACE(self->logger, KAA_ERR_NONE, "Going to add mandatory notification listener id '%u'", id);
```

Comment:
Please. move to the end and change message to:
KAA_LOG_TRACE(self->logger, KAA_ERR_NONE, "Added mandatory notification listener id '%u'", id);